scripts/convert-idl.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Transformer = require('webidl2js');
+
+const allProperties = require('../lib/allProperties.js');
+const allExtraProperties = require('../lib/allExtraProperties.js');
+const parsers = require('../lib/parsers.js');
+
+const srcDir = path.resolve(__dirname, '../src');
+const implDir = path.resolve(__dirname, '../lib');
+const outputDir = implDir;
+
+const propertyNames = Array.from(allProperties)
+  .concat(
+    Array.from(allExtraProperties).filter(prop => prop !== 'css-float' && !allProperties.has(prop))
+  )
+  .sort();
+
+const camelCasedAttributes = propertyNames.map(n => parsers.dashedToCamelCase(n));
+const webkitCasedAttributes = propertyNames
+  .filter(n => n.startsWith('-webkit-'))
+  .map(n => parsers.dashedToCamelCase(n.substring(1)));
```

Comment:
If you implement the full [CSS property to IDL attribute](https://drafts.csswg.org/cssom/#css-property-to-idl-attribute) algorithm as I commented above, the `.substring(1)` could be replaced by setting the _lowercase first_ flag.